contracts/wXTMBridge.sol

```diff
@@ -1,21 +1,27 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.22;
 
-import {EIP3009} from "./extensions/EIP3009.sol";
-import {ERC20Burnable} from "@openzeppelin/contracts/token/ERC20/extensions/ERC20Burnable.sol";
+import { EIP3009 } from "./extensions/EIP3009.sol";
+import { ERC20Burnable } from "@openzeppelin/contracts/token/ERC20/extensions/ERC20Burnable.sol";
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
 
-import {IwXTM} from "./interfaces/IwXTM.sol";
+import { IwXTM } from "./interfaces/IwXTM.sol";
 
-contract wXTMBridge is EIP3009 {
-    address private immutable wXTM;
+contract wXTMBridge is EIP3009, Ownable {
+    address private wXTM;
 
     event TokensMinted(address indexed to, uint256 indexed amount, bytes32 indexed authorizationId);
     event TokensBurned(address indexed from, uint256 indexed amount, bytes32 indexed authorizationId);
 
-    constructor(address _wXTM, string memory version) EIP3009("wXTMBridge", version) {
+    constructor(address _wXTM) Ownable(msg.sender) {
```

Comment:
probably instead of msg.sender it's more convenient to pass in the delegate (which will be a multisig). It would not be practical to deploy this contract directly from multisig